mass_editing/migrations/14.0.1.0.0/post-migrate.py

```diff
@@ -0,0 +1,38 @@
+# Copyright (C) 2020 - Iván Todorovich <ivan.todorovich@gmail.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from openupgradelib import openupgrade
+
+
+@openupgrade.migrate()
+def migrate(env, version):
+    if openupgrade.table_exists(env.cr, "mass_editing"):
+        migrate_mass_editing(env)
+
+
+def migrate_mass_editing(env):
+    """ Migrates mass.editing to ir.actions.servere """
+    # Remove previous context actions
+    env["ir.actions.act_window"].search(
+        [("res_model", "=", "mass.editing.wizard")]
+    ).unlink()
+    # Create new context actions
+    env.cr.execute(
+        """
+        SELECT id FROM ir_act_server
+        WHERE state = 'mass_edit'
+        AND __legacy_ref_ir_act_window_id__ IS NOT NULL
+        """
+    )
+    server_action_ids = [r[0] for r in env.cr.fetchall()]
+    if server_action_ids:
+        env["ir.actions.server"].browse(server_action_ids).create_action()
+    # Cleanup
+    openupgrade.logged_query(
```

Comment:
Generaly such cleanup is done after, with database_cleanup module.

Reply:
Yes but these are temporary columns created by this script, so I thought it's better to remove them.
The do not belong to previous fields, so they have no use whatsoever